module7_selenium/src/test/java/page/AbstractPage.java

```diff
@@ -0,0 +1,12 @@
+package page;
+
+import org.openqa.selenium.WebDriver;
+
+public abstract class AbstractPage {
+    protected static final String XPATH_PATTERN = "//div[contains(text(),'%s')]";
```

Comment:
Abstract page is not the place for this constant.
Also `XPATH_PATTERN` is not the clear name.